build.gradle

```diff
@@ -126,7 +126,7 @@ ext {
   userMaxTestRetryFailures = project.hasProperty('maxTestRetryFailures') ? maxTestRetryFailures.toInteger() : 0
 
   skipSigning = project.hasProperty('skipSigning') && skipSigning.toBoolean()
-  shouldSign = !skipSigning && !version.endsWith("SNAPSHOT") && !version.contains("-alpha") && project.gradle.startParameter.taskNames.any { it.contains("upload") }
+  shouldSign = !skipSigning && !version.endsWith("SNAPSHOT") && !version.contains("-alpha") && !version.matches("[0-9]*\\.[0-9]*\\.[0-9]*-[0-9]*-ccs") && project.gradle.startParameter.taskNames.any { it.contains("upload") }
```

Comment:
This line is slightly different for the `-ce` version, right? Can we just make the regex work for both here to reduce the diff between the repos?

Reply:
i updated this to match either ccs or ce